README.md

```diff
@@ -1,5 +1,7 @@
 # ManageIQ::Api
-Short description and motivation.
+
+ManageIQ::Api is a plugin to be used in the [ManageIQ project][ManageIQ]. It forms
+the public JSON API for ManageIQ.
```

Comment:
s/JSON/REST/, not because it's inaccurate but because we refer to it as the REST API basically everywhere so why not here, too.